packages/core/src/components/logo/logo.scss

```diff
@@ -0,0 +1,66 @@
+@use "~dso-toolkit/src/variables/colors";
+@use "~dso-toolkit/src/variables/media-query-breakpoints";
+@use "~dso-toolkit/src/variables/typography";
+
+:host {
+  display: block;
+  font-family: typography.$font-family-base;
+}
+
+svg {
+  display: block;
+  max-width: 100%;
+}
+
+.logo_content_omgevings {
+  white-space: pre;
+  font-size: 25px;
```

Comment:
waarom geen `rem`? nu schaalt deze niet mee als de gebruiker een andere tekstgrootte heeft ingesteld. Readability trumps design ;)

Ook: is de `font-size` niet beter vanuit de `:host` te zetten? nu zie 3 x dezelfde opduiken.

Reply:
Ik had eerder de SVG tekstelementen om te stylen en voordat ik de CSS nakeek wilde ik eerst zeker weten dat de SVG elementen de styling op de juiste manier zouden overnemen. Ik wist niet hoe 'braaf' het zou zijn met het overerven (misschien een restje omdat ik jarenlang HTML e-mails heb gebouwd)